Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ocaml/opam:debian-11-ocaml-4.14@sha256:70bd0da74f68550d667cc1a8ae80661e694cd9ac614273b23c45f3fb8a9f3ac9 AS build
+RUN sudo apt-get update && sudo apt-get install libev-dev libffi-dev capnproto m4 pkg-config libsqlite3-dev libgmp-dev graphviz -y --no-install-recommends
+RUN cd ~/opam-repository && git pull origin -q master && git reset --hard ed966e5e82880cba7c483d3a5c1c8d4328c4f0c1 && opam update
```

Comment:
You can do a `fetch` instead of a `pull` since you're doing a hard reset afterwards

Reply:
Yes of course, this was copied from other Dockerfiles in ocluster that could use the same treatment.